excel/beginner/data-validation-filtering-grouping/data-validation.md

```diff
@@ -0,0 +1,100 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: must-know
+
+---
+
+# Data Validation
+
+---
+## Content
+
+Excel has a feature called "Data Validation" which you can use to force a particular input into a cell.
+
+For instance, you can make a cell be of type `date` and only allow actual date input or a date `after`, `before` or even `between` a certain date.
+
+To open `data validation` you can either go to(Google sheets):
+
+![how-to-open-data-validation](https://img.enkipro.com/8cd2e9b2913a5dc3f348d3b93c90c3ea.png)
+
+or right-click any cell and select `data validation` at the bottom.
+
+The difference between these two ways of opening `data validation` is if you right-click a cell, it will select that cell to be validated. However, if you open it like in the picture above, you will have to select the cells on your own.
+
+This will give you a popup window like so:
+
+![data-validation](https://img.enkipro.com/e8e5151a6a2d4745a003e7c1b888ef7f.png)
+
+To select a cell click on `Cell range` and click on a cell. 
+
+To select a range of cells starting from a cell, click on a cell and drag in any direction to select a range of cells you want to use this validation on. 
+
+Under "Criteria" there are several options:
+
+![criteria-options](https://img.enkipro.com/ba4bb0f6f2fd08fb155c92ae0365a658.png)
+
+And each `option` has a different set of `sub-options` that will appear on the right side after choosing one. 

```

Comment:
"option" and "sub-options" aren't special words in any way, right?

```suggestion
Each option has a different set of sub-options that will appear on the right side after choosing one. 

```

Reply:
Right